matlab/polar-traversal/approximatelyEqual.m

```diff
@@ -0,0 +1,8 @@
+function [check] = approximatelyEqual(a, b, absEpsilon, relEpsilon)
+% https://www.learncpp.com/cpp-tutorial/relational-operators-and-floating-point-comparisons/
+    diff = abs(a-b);
+    if diff <= absEpsilon
+        check = true;
```

Comment:
Did you mean to return after this?

Reply:
Probably not; need conditional for odd case.

Reply:
yes!